src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4598,14 +4632,14 @@ address MacroAssembler::zero_words(Register ptr, Register cnt) {
     RuntimeAddress zero_blocks(StubRoutines::riscv::zero_blocks());
     assert(zero_blocks.target() != nullptr, "zero_blocks stub has not been generated");
     if (StubRoutines::riscv::complete()) {
-      address tpc = trampoline_call(zero_blocks);
+      address tpc = patchable_far_call(zero_blocks);
       if (tpc == nullptr) {
         DEBUG_ONLY(reset_labels(around));
         postcond(pc() == badAddress);
         return nullptr;
       }
     } else {
-      jump_link(zero_blocks, t0);
+      rt_call(zero_blocks.target(), t0);
```

Comment:
Maybe simply: `rt_call(zero_blocks.target());` as `t0` is the default temp register for `rt_call`.

Reply:
ok